docs/src/components/home/GoldSponsors.tsx

```diff
@@ -17,47 +17,47 @@ const GOLDs = [
     description: 'Enterprise-ready open-source software.',
     href: 'https://tidelift.com/subscription/pkg/npm-material-ui?utm_source=npm-material-ui&utm_medium=referral&utm_campaign=homepage',
   },
-  {
-    src: 'https://avatars.githubusercontent.com/u/1262264?size=40',
-    srcSet: 'https://avatars.githubusercontent.com/u/1262264?s=80 2x',
-    name: 'Text-em-all',
-    description: 'The easy way to message your group.',
-    href: 'https://www.text-em-all.com/?utm_source=MUI&utm_medium=referral&utm_content=homepage',
-  },
```

Comment:
They are still here, contributing $250/month: https://opencollective.com/callemall. They started the project, I think we should keep them.

x the other places